docker-compose.yml

```diff
@@ -12,8 +12,9 @@ services:
             - "5432"
 
     b2share:
-        image: eudatb2share/b2share:2.0.1
+        image: b2share:modes
```

Comment:
I'm not sure why the image tag has been modified in this way, I think we should keep the previous naming scheme. We need to put the version information here in any case.

Reply:
Right, to be changed.